nginx-proxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx
+
+RUN apt-get update && apt-get install -y netcat
```

Comment:
What are we doing with `netcat`?

Reply:
Ah https://github.com/futurelearn/deployment/pull/81/commits/f5a19f350b0f541dd0b92387980d29d0b66ff9a3#diff-fcc9f67616f16fa4f6a6196995e4c3ccR100

Reply:
Yeah, healthchecks.